src/app/sevices/user.service.ts

```diff
@@ -1,22 +1,14 @@
-import { Inject, Injectable, PLATFORM_ID } from "@angular/core";
-import { BehaviorSubject, Observable, ReplaySubject, Subject, take, tap } from "rxjs";
+import { Injectable } from "@angular/core";
+import { Observable } from "rxjs";
 import { User } from "../models/user.model";
 import { HttpClient } from "@angular/common/http";
 import { backend_users } from "../constants/endpoints";
```

Comment:
Why does it have different casing? In angular, most of the time we are using `camelCase`. for constants, it is usually `UPPER_SNAKE_CASE`